hrui/js/src/hrui.js

```diff
@@ -283,6 +284,19 @@
       .find('.crm-summary-block').show();
   }
 
+  /**
+   * Moves the "Extended Demographics" block after "Inline Custom Data"
+   * in the personal details tab
+   */
+  function repositionExtendedDemographicsInPersonalDetailsTab () {
```

Comment:
this function has a lot of code in common with `repositionInlineCustomDataBlockInPersonalDetailsTab`
```js
$(<selector-to-move>)
  .removeClass('crm-collapsible collapsed')
  .addClass('crm-summary-block')
  .insertAfter(<selector-to-target>)
  .find('.collapsible-title').hide().end()
  .find('.crm-summary-block').show();
```
meaning that we can make a more generic function that we can then use for both cases
```js
function manipulateDOMOfInlineCustomData () {
  // ...
  commonFunction('.Inline_Custom_Data', '.crm-s mmary-contactinfo-block');
  commonFunction('.Extended_Demographics', '.Inline_Custom_Data');
}
```
tricky part is to find a decent name for it, something like `repositionPersonalDetailsBlock`?

Reply:
Good idea 👍. I will refactor it.